src/js/tracker.js

```diff
@@ -901,7 +962,7 @@
 			 * @param string url
 			 */
 			setReferrerUrl: function (url) {
```

Comment:
@fblundun can you add some doco on setReferrerUrl in the user manual? There doesn't seem to be any.